### ISSUE TITLE
[AZE0] Heater programmator and remote heating fails after a night

### ISSUE THREAD

**Issue by Dav36** [dalathegreat/BatteryUpgrade-UserManual#26]
Hi,
I have a issue with the heating/cooling programmator and remote activation of it.

The problem comes after the leaf had been sleeping all the night. 
If i open and close the car, all is working ok... even if i wait 30 minutes...

I got the problem every mornings...

My leaf is swapped with 40kw battery and Dalla's can bridge.

Please help :)

**Comment by Dav36** [comment]
![Screenshot_2022-11-03-17-19-10-477_com Turbo3 Leaf_Spy_Pro](https://user-images.githubusercontent.com/117361584/199781893-0bc6482f-4962-4b38-9d5b-7e793a8094a6.jpg)
Here the leaspy pro at home after work

**Comment by Dav36** [comment]
I tried to clear DTC 
![Uploading Screenshot_2022-11-03-17-21-25-765_com.Turbo3.Leaf_Spy_Pro.jpg…]()

**Comment by Dav36** [comment]
![Screenshot_2022-11-03-17-21-25-765_com Turbo3 Leaf_Spy_Pro](https://user-images.githubusercontent.com/117361584/199783024-41d2747a-efca-45c0-9e62-25c898801412.jpg)

**Comment by dalathegreat** [comment]
@Dav36 , update to v3.12 , this will solve the issue. https://dalasevrepair.fi/wp-content/software/
Flashing instructions: https://www.youtube.com/watch?v=eLcNSo2Vn6U

**Comment by Dav36** [comment]
Thank you! 
--
Envoyé depuis l'application myMail pour Android vendredi, 04 novembre 2022, 01:47PM +01:00 de Daniel Öster  ***@***.*** :

***@***.*** , update to v3.12 , this will solve the issue.  https://dalasevrepair.fi/wp-content/software/

**Comment by Dav36** [comment]
Hello again, 
Canbridge flasher Ok. I get less faults on leafspy. 

But program heater still not working... 
I have programmed it at 7h30 yesterday evening and check it this morning at 7h16 = heater off, at 7h34 = heater off...  (Leaf plugged or not) 

Do you have an idea about this ? 
Br

**Comment by dalathegreat** [comment]
Strange, I do remote heating requests just fine via OVMS.

Which year is your LEAF? ZE0/AZE0? JDM/EUDM/USDM?

**Comment by Dav36** [comment]
I am talking about the heating planner, not the remote heating. 
My leaf tekna is from 2015 AZE0. 
I have noticed that the problem occurs when the leaf is in deep sleep, because if i turn the leaf on then off, and program the heating planner to start 30 minutes after, it works well... 
Hope this help

**Comment by Dav36** [comment]
Here a picture off the heating planner
![IMG_20221122_233135](https://user-images.githubusercontent.com/117361584/204162469-ba6e7be9-f5b0-495f-be53-78cf34f4128b.jpg)

**Comment by dalathegreat** [comment]
Yeah sorry I don't have that feature on my testing LEAF (base model), so I cannot debug this further.

**Comment by Dav36** [comment]
Hi dala, 

I made another test this morning, when the leaf is plugged and in deep sleep mode (after a night of charge) , the remote control of the heater doesn't works too... 
I think the issue impact both functionalities... 

Here after my last leafspy faults

**Comment by Dav36** [comment]
Maybe i can give you some CAN logs ?

**Comment by dalathegreat** [comment]
I think I found the issue for this. Please contact me via email and I can send you a beta software to fix this @Dav36
